docs/markdown/snippets/executable_vs_module_defs.md

```diff
@@ -0,0 +1,4 @@
+## Executable gains vs_module_defs keyword
+
+This allows using a .def file to control which functions an Executable will
+expose to a SharedModule.
```

Comment:
We should use meson DSL names for this, not internal class names -- so, interlinks in the form of `[[executable]]` and `[[shared_module]]`.

Reply:
I've updated this locally.